### PR TITLE
initial fastlane structure

### PR DESCRIPTION
**Pull request by IzzySoft**
Heard you need this, so I thought I send you what's set up with my repo. Be welcome to build on it and use my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) for guidance :wink:

### CODE REVIEW

**Comment by alessioC42** [comment]
Thank you. Helps a lot!

**Review by alessioC42 — APPROVED** [review]

File: fastlane/metadata/android/en-US/images/icon.png
Comment (IzzySoft) [inline fastlane/metadata/android/en-US/images/icon.png:1]: Oof. Please keep in mind that this is an icon, not a poster – there are limits to the icon size, see my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688), quote:

* `icon.png`: minimum 48x48, max 512x512 px

**Comment by IzzySoft** [comment]
Thanks for merging! Enabled that for my repo now, too, and pulled in the updated fulldesc (I missed the renaming of the app, ouch). On the next release, icon and screenshots will be pulled along as well, so please make sure to have the icon fixed by then (ah well, if not, my updater will take care for that – but still… :wink:)

File: fastlane/metadata/android/en-US/images/icon.png
Comment (alessioC42) [reply fastlane/metadata/android/en-US/images/icon.png:1]: Yeah, 3200x3200 might be a bit too big xD. I reduced it to 512x512

**Pull request merged**